boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -132,10 +112,37 @@ etc).  It might be necessary to change the flash frequency or the flash
 mode; please refer to the `esptool documentation`_ for guidance on these
 settings.
 
-If ``ESP_TOOL`` is set to "espidf", the `esptool.py`_ script found within
-ESP-IDF will be used.  Otherwise, this variable is handled as a path to
-the tool.
+All flashing options are now handled by the ``west`` tool, to flash with custom
```

Comment:
How about:

    All flashing options are now handled by the :ref:`west` tool, including flashing
    with custom options such as a different serial port.  The ``west`` tool supports
    specific options for the ESP32 board, as listed here:

I notice that there's no mention of these ESP32-specific options in the main west documentation (doc/west/index.rst).  That doesn't seem right?

Reply:
> I notice that there's no mention of these ESP32-specific options in the main west documentation (doc/west/index.rst). That doesn't seem right?

yeah, that is expected.